java-chassis-dependencies/springboot2/pom.xml

```diff
@@ -31,12 +31,16 @@
   <packaging>pom</packaging>
 
   <properties>
-    <spring.boot.version>2.1.2.RELEASE</spring.boot.version>
+    <!-- Dependency versions -->
+    <spring-boot.version>2.1.2.RELEASE</spring-boot.version>
```

Comment:
2.1.3.RELEASE is better

Reply:
Just in case, this PR is just for setting versions as properties, all versions are the same as previous. Updating version will be done after this PR.